src/provider/view/components/SettingsView/SettingsView.tsx

```diff
@@ -15,17 +17,23 @@ const mockFeed: Feed = {
 const mockFeeds = new Array(5).fill(mockFeed);
 
 export const SettingsView: React.FC = (props) => {
+    const storeApi = usePreduxStore();
```

Comment:
SettingsPanel/SettingsView should be now connected to the store and map the state (locked/muted) into the component to update properly.

Reply:
✅